src/langchain_google_memorystore_redis/__init__.py

```diff
@@ -11,3 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from langchain_google_memorystore_redis.redis_chat_message_history import MemorystoreChatMessageHistory
```

Comment:
should we name the file "chat_message_history.py". We already have redis in the package name to differentiate the engine. I would imagine when we started integrating with a different engine say memcached (if ever :-)), we would create a new package name like langchain_google_memorystore_memcached. What do you think?